lib/Timber.php

```diff
@@ -215,6 +216,65 @@ public static function get_term( $term, $taxonomy = 'post_tag', $TermClass = 'Ti
 		return TermGetter::get_term($term, $taxonomy, $TermClass);
 	}
 
+	/* User Retrieval
+	================================ */
+
+	/**
+	 * Get one or more users as a collection.
+	 * @api
+	 * @param array   $query The params for the WP_User_Query to performed
+	 * @param array   $options optional; none are currently supported
+	 * @return mixed
+	 */
+	public static function get_users( array $query = [], array $options = [] ) : Iterable {
```

Comment:
Should we really add an `$options` parameter if there are no options supported yet?

Reply:
Not sure about this one. This was back when I thought we'd return a `UserCollection`, so adding such an option might be imminent. So not as necessary. However, it does still come down to what constitutes a "breaking change" and the kinds of use-cases we explicitly support.

If don't ship with `$options` but then add it later, then someone could literally do `MyClass extends Timber` and override this method with single arity on 2.0 and then encounter a breakage just by updating to a minor version when we add the param. So that's technically a breaking change, even if we can't imagine why someone would do that instead of just using the filters we provide. But as long as there's a possibility, I think it might be good to signal that intention and leave the door open.

Curious to hear from @jarednova here.

Reply:
So it's basically like no options yet, but (perhaps) options to come? A good case could be made either way, but it seems like it could be helpful in remembering for when options get added this is where they go in order to match patterns seen elsewhere in the code base. Do I have that right?

Reply:
Yes.

Reply:
I didn’t think about the breaking change if someone extends `Timber\Timber`. Under these circumstances, it makes total sense. And even if we don’t have a use case now, there can very well be a use case in the future, as we have learned time and time again while developing and maintaining Timber.

Let’s leave it, then! 👍